apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -370,6 +370,26 @@
                             "minimum" : 0
                         }
                     }
+                },
+                "protocol_beneficiary_address" : {
+                    "description" :
+                    "Public key of beneficiary account that belongs to protocol maintainers.",
+                    "type" : "string",
+                    "example" : "ak_DummyPubKeyDoNotEverUse999999999999999999999999999",
+                    "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*$"
+                },
+                "protocol_beneficiary_enabled" : {
+                    "description" :
+                    "If true, the node will split rewards and send part to protocol_beneficiary_address",
+                    "type" : "boolean"
```

Comment:
Could you please move default here - deleting `-define(PROTOCOL_BENEFICIARY_ENABLED, true).`?

In this way, the default value would be clearly documented to the users, while still keeping only one canonical location for the default.

Reply:
I propose to keep in both places, similarly to e.g. EXPECTED_BLOCK_MINE_RATE_MINUTES

Reply:
Provided we keep them in sync, 👍

Reply:
@lucafavatella my concern is that the config file is very obscure from developer's point of view vs having them in clear view in arc_governance. If we go config, lets do it as refactoring and move all values.

Reply:
(by obscure I mean, for example, that IDE is not automatically suggesting value when reading the code)